package.json

```diff
@@ -66,6 +82,16 @@
                 {
                     "command": "aks.periscope",
                     "when": "view == kubernetes.cloudExplorer && viewItem =~ /aks\\.cluster/i"
+                },
+                {
+                    "command": "extension.BrowseCICDPipeline",
```

Comment:
should we use our extension name instead of `extension`? `azure-deploy.BrowseCICDPipeline`